### PR TITLE
Feature/traction 69

### PR DESCRIPTION
**Pull request by psharma-traction**
# Critical Changes

# Changes
contactCard : Included Label for Role on Contact Card in Home
ContactEditFormController : appiled try catch on dml operations and made picklistValues as map of label->  value
ContactEditFormControllerTest : updated for map related change
ContactsSelector: Included label for status as shown in toggle bar in home page
CredentialsSelector : included label for type in soql
contactEditForm : filteredPicklistValues updated for label and value and included typelabel
staffAvailability : Included staff label.
contactSupport : included translations.
Included helptext translation for custom fields and Account fields translation for consistency.

# Issues Closed
traction-69,70,71,374

### CODE REVIEW

File: force-app/main/default/lwc/staffAvailability/staffAvailability.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ export default class StaffAvailability extends LightningElement {
 		this.template.querySelector("c-app-spinner").displaySpinner(true);
 		getAssignmentData({contactId: this.recordId? this.recordId:null}).then(result => {
 			this.data = result;
-			this.staffStatus = getFieldValue(result.contact, 'Status__c', this.namespace);
+			this.staffStatus = getFieldValue(result.contact, 'StatusLabel', '');
```
Comment (cwico) [inline force-app/main/default/lwc/staffAvailability/staffAvailability.js:89]: could do:

this.staffStatus = result.contact.StatusLabel

NOTE: will have to be tested in the package to figure out whether it requires a namespace or now

File: force-app/main/default/classes/ContactEditFormController.cls [hunk 2]
```diff
@@ -96,13 +102,16 @@ public with sharing class ContactEditFormController {
...
 			throw new AuraHandledException('Wrong credential');
 		}
-
 		SecureDml insertObject = new SecureDml();
 		try{
 			insertObject.dmlInsert(credentialToInsert);
 		}catch (Exception e){
 			throw new AuraHandledException(JSON.serialize(e.getTypeName() + ': ' + e.getMessage() + ' - ' + e.getStackTraceString()));
 		}
+		/*Database.SaveResult saveResult = Database.insert(credentialToInsert, false);
```
Comment (pserquiz) [inline force-app/main/default/classes/ContactEditFormController.cls]: no more need for this part of the code? better to just delete ^^

File: force-app/main/default/classes/StaffAvailabilityController.cls [hunk 3]
```diff
@@ -106,8 +106,12 @@ public with sharing class StaffAvailabilityController {
 			} else {
 				contact[0].Status__c = Constants.NOT_AVAILABLE_STATUS;
 			}
-			SecureDml updateObject = new SecureDml();
-			updateObject.dmlUpdate(contact);
+			/*SecureDml updateObject = new SecureDml();
+			updateObject.dmlUpdate(contact);*/
+			Database.SaveResult saveResult = Database.update(contact[0], false);
```
Comment (cwico) [inline force-app/main/default/classes/StaffAvailabilityController.cls]: SecureDML checks for CRUD, why remove? Please use SecureDML the SF security team recommended we do this for the product.

**Review by cwico — APPROVED** [review]
Will approve for now, but please use SecureDML for any DML operations you will do.

**Review by pserquiz — APPROVED** [review]
Just do couple changes in the comments if it make sense before merging

File: force-app/main/default/classes/StaffAvailabilityController.cls (hunk 3 above)
Comment (ghost) [reply force-app/main/default/classes/StaffAvailabilityController.cls]: @psharma-traction  please see note above, secureDML is recommended.

Comment (psharma-traction) [reply force-app/main/default/classes/StaffAvailabilityController.cls]: updated with the SecureDML.

File: force-app/main/default/classes/ContactEditFormController.cls (hunk 2 above)
Comment (psharma-traction) [reply force-app/main/default/classes/ContactEditFormController.cls]: Removed comments

**Pull request merged**